File: app/src/main/java/com/example/algorithm/HashTestA.java
package com.example.algorithm;

import java.util.ArrayList;
import java.util.Arrays;

public class HashTestA {
    String Solution(String[] participant, String[] completion){

        Arrays.sort(participant);
        Arrays.sort(completion);
        int i;
        for (i=0; i<completion.length; i++){
            if (!participant[i].equals(completion[i])){
                return participant[i];
            }
        }

        return participant[i];
    }
}

File: app/src/main/java/com/example/algorithm/HashTestA.kt
package com.example.algorithm

class HashTestA {

    fun solution(a : Array<String>, b: Array<String>) : String{
        val temp = a.toMutableList()
        b.forEach {
            if (temp.contains(it)){
                temp.remove(it)
            }
        }

        return temp.first()
    }
}